x/marker/types/msgs_test.go

```diff
@@ -43,6 +45,11 @@ func TestAllMsgsGetSigners(t *testing.T) {
 		func(signer string) sdk.Msg { return &MsgSetAccountDataRequest{Signer: signer} },
 		func(signer string) sdk.Msg { return &MsgUpdateSendDenyListRequest{Authority: signer} },
 		func(signer string) sdk.Msg { return &MsgAddNetAssetValuesRequest{Administrator: signer} },
+		func(signer string) sdk.Msg { return &MsgSupplyDecreaseProposalRequest{Authority: signer} },
+		func(signer string) sdk.Msg { return &MsgChangeStatusProposalRequest{Authority: signer} },
+		func(signer string) sdk.Msg { return &MsgSetDenomMetadataProposalRequest{Authority: signer} },
+		func(signer string) sdk.Msg { return &MsgRemoveAdministratorProposalRequest{Authority: signer} },
+		func(signer string) sdk.Msg { return &MsgSetAdministratorProposalRequest{Authority: signer} },
```

Comment:
Missing entry for `MsgWithdrawEscrowProposalRequest`. Also, please order them the same as the `.proto` file.